src/qibolab/pulses.py

```diff
@@ -1802,8 +1813,19 @@ def get_qubit_pulses(self, *qubits):
 
         new_pc = PulseSequence()
         for pulse in self.pulses:
-            if pulse.qubit in qubits:
-                new_pc.add(pulse)
+            if not isinstance(pulse, CouplerFluxPulse):
+                if pulse.qubit in qubits:
+                    new_pc.add(pulse)
+        return new_pc
+
+    def get_coupler_pulses(self, *couplers):
+        """Returns a new PulseSequence containing only the pulses on a specific set of couplers."""
+
+        new_pc = PulseSequence()
+        for pulse in self.pulses:
+            if isinstance(pulse, CouplerFluxPulse):
+                if pulse.qubit in couplers:
```

Comment:
Do we really need a `CouplerFluxPulse`? Is that different from a `FluxPulse`?

Otherwise, 
```suggestion
                if pulse.qubit in couplers:
```
is already enough for me to do what is described by the function name and docstring.

Reply:
If we name `couplers: [0,1,3, 4]` and` qubits: [0,1,2,3,4]` which would be a common thing to happen, and as in the previous comment, `pulse.qubit` only returns the name pulses will almost all times be assigned to the couplers.

Reply:
But then this is a different problem, concerning the name used in `pulse.qubit`, and addressing qubits in general.

In case there are these kinds of problems, please raise the issue, and try avoiding workarounds.
The problem is that these shortcuts are going to stratify, layer over layer, and reverting them will become increasingly difficult.
Of course there is a trade-off, we also have to merge PRs and make changes. But if you need a shortcut, at least raise an issue where the shortcut is documented (or document on an existing one).

Reply:
Done #605.

Reply:
Thanks @stavros11